HxCFloppyEmulator_software/sources/thirdpartylibs/fltk/.gitignore

```diff
@@ -0,0 +1,2 @@
+fltk-1.x.x*
```

Comment:
Also covers the `fltk-1.x.x-arm64/` / `fltk-1.x.x-x86_64/` directories I had on my Mac after building